js/bundle.go

```diff
@@ -307,6 +307,28 @@ func (b *Bundle) instantiate(logger logrus.FieldLogger, rt *goja.Runtime, init *
 	}
 	rt.Set("__ENV", env)
 	rt.Set("__VU", vuID)
+	rt.SetPromiseRejectionTracker(func(p *goja.Promise, op goja.PromiseRejectionOperation) {
+		// TODO this only handles the case where a promise get's rejected without having a reject handler added.
+		// But the other case where for an already rejected promise (one that this call has already handled)
+		// there is now a reject handler attached is not handled.
+		//
+		// More complete implementation might need to wait for some time, the end of an iteration,
+		// some other event before logging a "rejection" that never got "handled"
```

Comment:
>waiting some time or end of iteration seems like a good idea but it probably needs an "OOM" prevention to not wait on ... **1milltion** unhandled rejections or something like that.

So, is this a real order of magnitude in the case we would track/registry all the "not yet handled" rejection? Seems so high to me but I expect to be my fault in terms of JavaScript world knowledge.